public/netlify.toml

```diff
@@ -0,0 +1,10 @@
+[build]
+publish = "build"  
+command = "npm run build" 
+ignore = "git diff --quiet HEAD^ HEAD README.md"
+
+
+[[redirects]]
```

Comment:
Please remove the redirects as we already defined them in [public/_redirects](https://github.com/FOSS-Community/fosscu.org/blob/main/public/_redirects)